en/content/visiting.html

```diff
@@ -491,3 +490,21 @@ <h2 id="visitor-targets">Visitor targets</h2>
     </tr>
   </tbody>
 </table>
+
+
+
+<h2 id="troubleshooting">Troubleshooting</h2>
+<p>
+  Normally all documents share the same <a href="buckets.html#bucket-space">bucket space</a>
+  - documents for multiple schemas are co-located.
```

Comment:
Typography nerd nitpick: consider replacing with non-spaced em-dash 👀 (macOS shortcut: `shift` `option` `-`)
```suggestion
  Normally all documents share the same <a href="buckets.html#bucket-space">bucket space</a>—documents
  for multiple schemas are co-located.
```

Reply:
I saw that coming :-)